src/core/appdir.cpp

```diff
@@ -119,13 +119,18 @@ namespace linuxdeploy {
                     // platform dependent implementation of copyright files deployment
                     std::shared_ptr<copyright::ICopyrightFilesManager> copyrightFilesManager;
 
+                    // Enviroment variable delimiter
+                    char envDelimiter = ';';
+
                     // decides whether copyright files deployment is performed
                     bool disableCopyrightFilesDeployment = false;
 
                 public:
                 PrivateData() : copyOperationsStorage(), stripOperations(), setElfRPathOperations(), visitedFiles(), appDirPath(), excludeLibraryPatterns() {
                         copyrightFilesManager = copyright::ICopyrightFilesManager::getInstance();
-                    };
+                        util::misc::charFromEnv("LINUXDEPLOY_DELIMITER", envDelimiter);
```

Comment:
Why even provide the option for a custom delimiter? That just complicates the code but does not add any value. Plus, linuxdeploy _already_ hardcodes delimiters elsewhere.

Reply:
removed